.yarnrc.yml

```diff
@@ -0,0 +1,10 @@
+npmRegistryServer: https://registry.npmjs.org
+npmPublishRegistry: https://registry.npmjs.org
```

Comment:
The default (`https://registry.yarnpkg.com`) is just a CNAME to `https://registry.npmjs.com` (https://yarnpkg.com/getting-started/qa#why-registryyarnpkgcom-does-facebook-track-us)